ic-asset/src/upload.rs

```diff
@@ -40,6 +40,7 @@ pub async fn upload(
         &batch_id,
         asset_locations,
         &container_assets,
+        vec![],
```

Comment:
wasn't sure about that

so how would that work? ic_asset::upload can be called on a single file - should we scan the directory for `.ic-assets.json` as well?

I was thinking about using `Default` values when `upload` is called, see https://github.com/dfinity/agent-rs/blob/73dd8c931c43c2e59ac160a41af67e72d4ec9bd7/ic-asset/src/plumbing.rs#L243

Reply:
We might have to start with defaults for upload.  It's more difficult, because we need to know which parent directories might also contain .ic-asset.json.